lib/bundler/installer/standalone.rb

```diff
@@ -15,24 +15,31 @@ def generate
         file.puts "ruby_engine = defined?(RUBY_ENGINE) ? RUBY_ENGINE : 'ruby'"
         file.puts "ruby_version = RbConfig::CONFIG[\"ruby_version\"]"
         file.puts "path = File.expand_path('..', __FILE__)"
-        paths.each do |path|
-          file.puts %($:.unshift "\#{path}/#{path}")
-        end
-      end
-    end
 
-  private
+        @specs.each do |spec|
+          next if spec.name == "bundler"
+
+          # Ensure that all `require_paths` exist.
```

Comment:
I'm not sure if this code is important, but a test fails without it so I kept it.